### PR TITLE
Delete CVE-2022-22122

### PR DESCRIPTION
**Pull request by hagaiwech**
Committed by: Hagai Wechsler

### CODE REVIEW

File: 2022/22xxx/CVE-2022-22122.json [hunk 1]
```diff
@@ -1,196 +1 @@
-{
```
Comment (cve-team) [inline 2022/22xxx/CVE-2022-22122.json:1]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/22xxx/CVE-2022-22122.json:1]: ('Expecting value: line 2 column 1 (char 1)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by hagaiwech** [comment]
@cve-team This vulnerability was assigned two CVEs by different CNAs.
We as WhiteSource CNA wish to delete our CVE from the list, to prevent duplicates.
Please tell us how to change the JSON file to hold a DELETED CVE

**Comment by anthnysingleton** [comment]
Hello @hagaiwech,

Can you resolve the merge conflict for this PR? Also deleting the contents of the file is not the proper action to take. You should create a JSON file that validates against the [CVE_JSON_4.0_min_reject.schema](https://github.com/CVEProject/cve-schema/blob/master/schema/v4.0/CVE_JSON_4.0_min_reject.schema).

You can also use the following Description template for the Description of the record.

`** REJECT **
DO NOT USE THIS CANDIDATE NUMBER.  ConsultIDs: [ID].  Reason: This candidate is a reservation duplicate of [ID].  Notes: All CVE users should reference [ID] instead of this candidate.  All references and descriptions in this candidate have been removed to prevent accidental usage.`

Once these corrections have been made we can proceed with processing this request. 

Thanks,
CVE Content Team

**Comment by hagaiwech** [comment]
Hi @anthnysingleton the JSON is now corrected. Please review our latest changes, Thanks.

**Comment by anthnysingleton** [comment]
Hello @hagaiwech,

Once you resolve the merge conflict for this PR we can process the request.

Thanks,
CVE Content Team

**Comment by hagaiwech** [comment]
@anthnysingleton resolved.

**Pull request merged**